cartographer_rviz/cartographer_rviz/drawable_submap.h

```diff
@@ -46,9 +47,10 @@ class DrawableSubmap : public QObject {
  public:
   // Each submap is identified by a 'trajectory_id' plus a 'submap_index'.
   // 'scene_manager' is the Ogre scene manager to which to add a node.
-  DrawableSubmap(int trajectory_id, int submap_index,
-                 Ogre::SceneManager* scene_manager,
-                 ::rviz::Property* submap_category, const bool visible);
+  DrawableSubmap(const int trajectory_id, const int submap_index,
```

Comment:
const int -> int. The const is an implementation detail for copied and has no meaning in the interface, which (ideally) should be the only thing exposed in the header.